### ISSUE TITLE
The SHA of the latest Windows Master nightlies version available is outdated

### ISSUE THREAD

**Issue by agiudiceandrea** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#938]
## Description

The SHA of the latest Windows Master nightlies version available listed in the "All Download" page is outdated.

The currently listed SHA in the page is:

Master nightlies SHA: [5ebf9e185e](https://github.com/qgis/QGIS/commit/5ebf9e185e)

The actual SHA is:
Master nightlies SHA: [8498c5549f](https://github.com/qgis/QGIS/commit/8498c5549f)

Page URL: https://qgis.org/en/site/forusers/alldownloads.html#windows

**Comment by rduivenvoorde** [comment]
Mmm, thought to fix it in the rst of the page... but it is inserted probably based on some javascript

```
.. [1] Latest release nightlies SHA: :raw-html:`<span id="qgis-rel-dev"> </span>`
.. [2] Latest long-term release nightlies SHA: :raw-html:`<span id="qgis-ltr-dev"> </span>`
.. [3] Master nightlies SHA: :raw-html:`<span id="qgis-dev"> </span>`
.. [4] Nightlies are debug builds (including debugging output)
..
```

Looking in Dev panel of browser, there are 3 url's being retrieved (and inserted in those places):

https://qgis.org/downloads/qgis-ltr-dev.sha
https://qgis.org/downloads/qgis-rel-dev.sha
https://qgis.org/downloads/qgis-dev.sha

The last one shows me `104:5ebf9e185e` in a browser (and 304/not-modified)

@jef-n is there something going wrong with how are these files created maybe?

**Comment by jef-n** [comment]
Fixed in 855f9095f50b99d9f07d44674e231fbb8a79aeee
